src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,65 @@
+import React, { useMemo } from 'react';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+
+type Props = {
+  people: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const getSortSearchParams = (sortField: string) => {
+    if (sortField === sort) {
+      return order === 'desc'
+        ? { sort: null, order: null }
+        : { sort: sortField, order: 'desc' };
+    }
+
+    return { sort: sortField, order: null };
+  };
+
+  const getArrowDirection = (sortField: string) => {
+    if (sortField === sort) {
+      return order === 'desc' ? 'fa-sort-down' : 'fa-sort-up';
+    }
+
+    return 'fa-sort';
+  };
```

Comment:
this can be moved outside of a component

Reply:
I use searchParams inside these functions. If they are moved from the component - I will be forced to pass more parameters.